server.js

```diff
@@ -2618,6 +2618,47 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// dotnet-status integration.
+camp.route(/^\/dotnetstatus\/(.+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var csproj = match[1];
+  var format = match[2];
+  var url = 'http://dotnet-status.com/api/status/' + csproj + '/';
+  var badgeData = getBadgeData('dotnetstatus', data);
+  request(url, function (err, res, buffer) {
+    if (err != null) {
+      badgeData.text[1] = 'inaccessible';
+      sendBadge(format, badgeData);
+      return;
+    }
+    if (res.statusCode === 404) {
+      console.log(404);
+      badgeData.text[1] = 'not found';
+      sendBadge(format, badgeData);
+      return;
+    }
+    try {
+      var data = JSON.parse(buffer);
+
+      badgeData.text[0] = 'up-to-date';
+      var upToDate = data.upToDate;
+      if (upToDate) {
+        badgeData.text[1] = 'true';
+        badgeData.colorscheme = 'blue';
+      } else {
+        badgeData.text[1] = 'false';
```

Comment:
Instead of saying `true` or `false`, be consistent with the other dependency tracking badges (like david-dm). The left side should sy "dependencies", and the right side should say something like "up to date", "out of date", or "insecure": https://github.com/badges/shields/blob/master/server.js#L2577